theme/templates/resource-landing-page/title-section.html

```diff
@@ -102,6 +84,35 @@
     {% endif %}
 {% endif %}
 
+{# ======= Check MODFLOW Model Instance content files =======#}
+{% if cm.resource_type == "MODFLOWModelInstanceResource" %}
+    {% if cm.check_content_files %}
+        <div class="col-sm-12">
+            <div id="missing-metadata-or-file" class="alert persistent {% if just_created %}alert-success{% else %}alert-warning{% endif %} alert-dismissible" role="alert">
+                <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span
+                        aria-hidden="false">&times;</span></button>
+
+                    {% if cm.check_content_files == 'multiple_nam' %}
+                        <br>
```

Comment:
We should move this <br> tag outside the if statement and just have one above it since both if clauses have it at the beginning.